modules/data.land/R/ic_process.R

```diff
@@ -6,14 +6,15 @@
 ##' @param input Taken from settings$run$inputs. This should include id, path, and source
 ##' @param dir settings$database$dbfiles
 ##' @param overwrite Default = FALSE. whether to force ic_process to proceed
+##' @param site Current site information
```

Comment:
This line is defining a new argument to the function, but the argument has't been added to the function itself